acados/sim/sim_erk_integrator.c

```diff
@@ -698,7 +698,7 @@ int sim_erk(void *config_, sim_in *in, sim_out *out, void *opts_, void *mem_, vo
                     }
                 }
                 acados_tic(&timer_ad);
-                if (opts->sens_hess)
```

Comment:
Same comment as above. I am afraid you may have some conflicts with @FreyJo 's PR down the road

Reply:
@roversch please try if your stuff works with my erk version. It looks like the function `expl_ode_hes` is not used anywhere with your modification. But as it was added to the model with some intention, I think it is more likely that we are using `expl_vde_adj` not as intended, i.e. to propagate the hessian..

Reply:
same comment as above